### PR TITLE
Add arp-driver recipe

### PR DESCRIPTION
**Pull request by TSAnsari**
Signed-off-by: tansari <tansari@luxoft.com>

### CODE REVIEW

**Review by thenor — APPROVED** [review]

File: recipes-kernel/arp-driver/arp-driver_git.bb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+inherit module
+
+SRCREV = "96d8815cd1dc6a47e091d2865939e62c132caf1d"
+
+SRC_URI = "git://github.com/TSAnsari/arp-driver.git;protocol=https;branch=master"
```
Comment (erikboto) [inline recipes-kernel/arp-driver/arp-driver_git.bb]: Why would the code be hosted on your private fork? Also the SRCREV doesn't seems to exist.

**Comment by sashko** [comment]
In addition to Erik's comment, I am not sure I understand what the "ARP driver kernel module" is -- isn't it supposed to be a bunch of kernel modules with different drivers needed for the ARP?

**Review by sashko — CHANGES_REQUESTED** [review]
recipe is broken

File: recipes-kernel/arp-driver/arp-driver_git.bb (hunk 1 above)
Comment (TSAnsari) [reply recipes-kernel/arp-driver/arp-driver_git.bb]: The revision of arp-driver repo which this points to is not yet delivered, so I used my private fork to test this. Once the arp-driver changes are delivered to master, then I will update this to point to that revision. Will definitely not push with this pointing to private fork.

Comment (frznlogic) [reply recipes-kernel/arp-driver/arp-driver_git.bb]: Please add a WIP: prefix to the PR in that case to indicate that it is Work In Progress.

**Comment by TSAnsari** [comment]
@sashko This is how ARP team has structured the drivers at the moment. All exists under one module name arp-driver. If it changes in the future then we need to create separate recipies for each of them. Hope thats what you mean?

**Pull request merged**